### DISCUSSION TITLE
Is it normal the podman machine has to be rebuilt on minor release

### DISCUSSION THREAD

**Discussion by johnnykang** [containers/podman#22877] [General]
I noticed that whenever there is a minor release,  for example, 5.0.x -> 5.1.x

All the existing containers would not work and I have to rebuild the podman machine then all the containers after the minor version upgrade.

Is that normal? The error i was been getting is it can't find the socks  connection despite the podman machine WSL is running.

I am on Win11 x64 WSL2.

**Comment by Luap99** [comment]
Yes and no, to get new updates inside the machine you have to recreate or update inside the VM until we have a proper update logic in place.

But as far as connections and stuff no that should just continue to work. You likely just fell victim to https://github.com/containers/common/issues/2025 here which will get fixed in 5.1.1 and then likely requires another machine recreation or you just move the file from the wrong location to the correct one then.

**Reply by johnnykang** [reply]
> You likely just fell victim to [containers/common#2025](https://github.com/containers/common/issues/2025)

Spot on. That's exact what had happened to me when I read the link you posted.

Thanks for the quick reply and insight.
